test/server_test.go

```diff
@@ -269,7 +269,7 @@ func TestTimeout(t *testing.T) {
 	// Try with a global client setting
 	c = c.WithTimeout(10 * time.Millisecond)
 	err = c.HealthCheck(context.Background())
-	assert.Error(t, err)
+	require.Error(t, err)
 	assert.Equal(t, "context deadline exceeded", err.Error())
 
 }
```

Comment:
added a test and confirmed that it failed before the change: https://github.com/Clever/wag/pull/52/commits/72dcb3eb93e3e7ceeda544c936d6e49e163c7609